backend-server/src/main/java/org/dice/ida/action/process/ActionMapper.java

```diff
@@ -0,0 +1,41 @@
+package org.dice.ida.action.process;
+
+import java.util.Map;
+
+import org.dice.ida.action.def.Action;
+import org.dice.ida.action.def.DefaultAction;
+import org.dice.ida.action.def.LoadDataAction;
+import org.dice.ida.action.def.SimpleTextAction;
+import org.dice.ida.model.Intent;
+public class ActionMapper {
```

Comment:
Codacy found an issue: [The utility class name 'ActionMapper' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/gh/dice-group/IDA/pullRequest?prid=6037853)